app/views/static/sandbox.scala.html

```diff
@@ -0,0 +1,28 @@
+@*****************************************************************************
+ * Page for testing XPath/XQuery expressions against target resources        *
+ * Useful for authoring Finders                                              *
+ * Copyright (c) 2015 MIT Libraries                                          *
+ *****************************************************************************@
+@(sbForm: Form[(String, String)], results: List[String])@*(implicit hubContext: HubContext)*@
+@import helper._
+@layout.main("Sandbox - TopicHub") {
+  <div class="container">
```

Comment:
Changing this to `<div class="container-fluid">` prevents it from sliding behind the top navigation.